herbert_core/utilities/hdf_nexus/find_root_nexus_dir.m

```diff
@@ -61,26 +72,28 @@
     return;
 end
 if(n_nx_entries>1)
-    error('ISIS_UTILITES:invalid_argument',' found multiple nxspe folders in file %s but this function does not currently support multiple nxspe folders',hdf_fileName)
+    if ~test_mode
```

Comment:
why is this support implemented in test mode but not production mode?

Reply:
It was discussion, when nxspe file format was introduced, about the possibility to store more then one nxspe data block in a file. This was not rejected but was not implemented. May be we may want to enable this possibility for our files (nxsqw?) In this case, this option should be enabled and tested more carefully but meanwhile -- let it be there but disabled.